lib/plugin.js

```diff
@@ -115,7 +115,13 @@ internals.schema.strategy = Joi.object({
         timeSkewSec: Joi.number().integer().min(0).default(0)
     })
         .when('.validate', { is: Joi.not(false), then: Joi.allow(false) })
-        .required()
+        .required(),
+
+    headerName: Joi.alternatives()
```

Comment:
Ideally the string would be restricted to valid header names using an appropriate regex.